.github/ISSUE_TEMPLATE/doc_improvement.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Documentation improvement
+about: Create a report to help us improve the documentation
+title: ''
+labels: Documentation, help wanted, good first issue
```

Comment:
For some reason this needs to be lower case for this to work. I hope github is not case sensitive.

```suggestion
labels: documentation, help wanted, good first issue
```

Reply:
I'm not sure about the good first issue though. Maybe easy by default, unless we change it?

Reply:
It was created through GitHub template creator. I should work.